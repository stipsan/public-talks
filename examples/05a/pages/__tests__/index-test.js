import { shallow } from 'enzyme'
import App from '../index'

it('should open and close', () => {
  const app = shallow(<App />)
  expect(app).toMatchSnapshot()
  expect(app.state().isOpen).toBe(false)

  app.find('button').simulate('click')
  expect(app).toMatchSnapshot()
  expect(app.state().isOpen).toBe(true)
})
