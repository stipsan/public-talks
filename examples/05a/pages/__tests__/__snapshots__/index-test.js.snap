// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should open and close 1`] = `
<div>
  <button
    onClick={[Function]}
  />
</div>
`;

exports[`should open and close 2`] = `
<div>
  <div>
    Modal content
  </div>
  <button
    onClick={[Function]}
  />
</div>
`;
